src/Form/MediaBundleDeleteForm.php

```diff
@@ -28,7 +28,7 @@ public function getQuestion() {
    * {@inheritdoc}
    */
   public function getCancelUrl() {
-    return new Url('media.overview_bundles');
+    return new Url('entity.media_bundle.collection');
```

Comment:
You can use the link template now with return $this->entity->urlInfo('collection'). Also, make sure you follow https://www.drupal.org/node/1728804